File: 48-rotate-image/48-rotate-image.py
class Solution:
    def rotate(self, a: List[List[int]]) -> None:
        """
        Do not return anything, modify matrix in-place instead.
        """
        a.reverse()
        for i in range(len(a)):
            for j in range(i):
                a[i][j],a[j][i]=a[j][i],a[i][j]

File: 48-rotate-image/48-rotate-image.cpp
class Solution {
public:
    void rotate(vector<vector<int>>& a) {
        reverse(a.begin(),a.end());
        for(int i=0;i<a.size();i++){
            for(int j=0;j<i;j++){
                swap(a[i][j],a[j][i]);
            }
        }
    }
};